chapter_deep-learning-computation/model-construction_vn.md

```diff
@@ -60,33 +60,42 @@ Fortunately, due to some behind-the-scenes magic supplied by the `autograd` pack
 (introduced in :numref:`chap_preliminaries`) when defining our own `Block`, we only need to worry about parameters and the `forward` function.
 -->
 
-*dịch đoạn phía trên*
+Để lập trình các mạng phức tạp này, ta sẽ giới thiệu khái niệm *khối* trong mạng nơ-ron.
+Một khối có thể mô tả một tầng duy nhất, một thành phần đa tầng hoặc toàn bộ một mô hình! <!-- Reviewer thấy từ này hợp lý hơn 'thành phần'('component') thì thay giúp mình nhé, thanks-->
+Dưới góc nhìn phần mềm, một `Block` (Khối) là một *lớp*.
+Bất kỳ một lớp con nào của `Block` đều phải định nghĩa phương thức `forward` để chuyển hóa đầu vào thành đầu ra và phải lưu trữ mọi tham số cần thiết.
+Lưu ý rằng một vài `Block` sẽ không yêu cầu bất kỳ tham số nào cả!
+Ngoài ra, một `Block` phải sở hữu một phương thức `backward` cho mục đích tính toán gradient.
+May mắn thay, nhờ có sự trợ giúp của gói `autograd` (được giới thiệu trong :numref:`chap_preliminaries`) nên khi định nghĩa `Block`, ta chỉ cần quan tâm đến các các tham số và hàm `forward`. 
+<!-- Cụm 'some behind-the-scenes magic' mình thấy khá hay nhưng dịch sang tiếng Việt hơi thô nên để đơn giản là 'trợ giúp', reviewer có cách dịch nào sát với sách hơn thì thay giúp mình.
+Mình không dịch các từ 'Block' (khi được viết hoa), 'forward', 'backward' vì thấy ở phần sau các từ này đ xuất hiện trong code, nên nếu dịch có thể gây nhầm lẫn, không biết mình hiểu vậy đúng không-->
```

Comment:
:+1:

Reply:
Yeah, rule of thumb là từ nào để trong backtick ```` thì không dịch.

Reply:
Ps: backtick khác dấu nháy 'đơn' nghen.

Reply:
Cám ơn mọi người nhé!